extensions/blocks/podcast-player/editor.scss

```diff
@@ -20,3 +20,20 @@
 	bottom: 0;
 	opacity: 0;
 }
+
+// Placeholder styles
+.jetpack-podcast-player__placeholder {
+	.components-base-control,
+	.components-base-control__field {
```

Comment:
While we were using the same components as the RSS block and the link below the input from the core embed block, not all styles are shared. Both these blocks add custom styles that are specific to these blocks but affect the visual appearance of the placeholder. This PR extracts the minimal necessary styles from both blocks.